src/Http/Controllers/AuthorizedAccessTokenController.php

```diff
@@ -59,5 +59,7 @@ public function destroy(Request $request, $tokenId)
         }
 
         $token->revoke();
+
+        return new Response('Token successfully revoked.', Response::HTTP_NO_CONTENT);
```

Comment:
`204` is intended to indicate that there is no body in the response.

See https://httpstatuses.com/204:

> A 204 response is terminated by the first empty line after the header fields because it cannot contain a message body.

```suggestion
        return new Response('', Response::HTTP_NO_CONTENT);
```

Reply:
You're correct. Fixed.